app/models/user.rb

```diff
@@ -1,4 +1,9 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable,
+         :confirmable
   has_many :posts, foreign_key: 'author_id'
   has_many :comments, foreign_key: 'author_id'
```

Comment:
- [ ] Please make sure that registration works probably here you have validation for the post_counter but in the signup, you don't so this will prevent the user from signing up please fix this.
![image](https://user-images.githubusercontent.com/74744787/195837180-6a700b6f-51f6-4741-91c4-39f155423f3b.png)
![image](https://user-images.githubusercontent.com/74744787/195837301-a3ba4f77-7913-4b2b-a7fe-72140ec668a3.png)

Reply:
### Hi, thanks for the review
I didn't get any error when I tried to create a new user, the ```posts_counter``` already has a default value, so, it does not need to be included in creating a new user. For example
- You can use the code below to create a new user
 ```
user = User.new(email: 'user@email.com', name: 'John Doe', bio: 'Developer', photo: 'https://www.w3schools.com/howto/img_avatar.png', password: 'password')
```
then
```
user.save
```

### Screenshot
![Screenshot from 2022-10-14 16-21-55](https://user-images.githubusercontent.com/78251927/195883430-45187279-2c3c-442a-9f3f-1ed44104f8a4.png)

- Click the link ```confirm my account``` to visit the login page and login

Reply:
Not actually check it you can't add a user 
![image](https://user-images.githubusercontent.com/74744787/195929213-9a52e2cc-501e-47f7-a879-0f38452c9a9c.png)

Reply:
![image](https://user-images.githubusercontent.com/74744787/195930033-e7cf8b7f-40bc-4c63-9690-b4e2eab038a3.png)

Reply:
![image](https://user-images.githubusercontent.com/74744787/195930103-cd26a43e-f4d8-4231-97ed-7a2d841a90fc.png)

Reply:
I can't sign up 
![image](https://user-images.githubusercontent.com/74744787/195977819-05d04a5f-1626-408a-97d4-9910d10129ac.png)

Reply:
you can see that the default value wasn't in the schema here 
![image](https://user-images.githubusercontent.com/74744787/195978921-4792d672-525e-437a-b998-ff3a00d7c68c.png)

Reply:
@abdulhamiid